rules/windows/process_creation/proc_creation_win_attrib_system_susp_paths.yml

```diff
@@ -4,10 +4,12 @@ related:
     - id: bb19e94c-59ae-4c15-8c12-c563d23fe52b
       type: derived
 status: experimental
-description: Detects usage of attrib with "+s" option to set suspicious script or executable as system files to hide them from users and make them unable to delete with simple rights. The rule limit the search to specific extensions and directories to avoid FP's
+description: |
+    Detects usage of attrib with "+s" option to set suspicious script or executable as system files to hide them from users and make them unable to delete with simple rights. The rule limit the search to specific extensions and directories to avoid FP's
```

Comment:
```suggestion
description: |
    Detects the usage of attrib with the "+s" option to set suspicious scripts or executables as system files to hide them from users and make them unable to be deleted with simple rights. The rule limits the search to specific extensions and directories to avoid FPs
```